math/FunMath.cpp

```diff
@@ -1 +1,10 @@
 #include "FunMath.h"
+#include <vector>
+
+
+std::vector<int> squaredvector(int squared) {
```

Comment:
- Add a function comment: what does it do, what are the parameters/return values
   - Make sure to follow our style guide for functions NamedLikeThis

Reply:
Added comment